src/components/posts/OrderPosts.js

```diff
@@ -21,46 +18,40 @@ function OrderPosts({
   return (
     <div className={classes.order}>
       <div className={classes.actions}>
-        {orderUpDown ? (
-          <button onClick={upHandler}>UP</button>
-        ) : (
-          <button onClick={upHandler} style={{ backgroundColor: "blue" }}>
-            UP
-          </button>
-        )}
-      </div>
-
-      {authorDate ? (
-        <>
-          <div>
-            <button onClick={dateHandler} style={{ backgroundColor: "blue" }}>
+        {authorDate ? (
+          <>
+            <button
+              onClick={dateHandler}
+              style={{ backgroundColor: "#ba7a92" }}
+            >
               Date
             </button>
-          </div>
-          <div>
             <button onClick={authorHandler}>Author</button>
-          </div>
-        </>
-      ) : (
-        <>
-          <div>
+          </>
+        ) : (
+          <>
             <button onClick={dateHandler}>Date</button>
-          </div>
-          <div>
-            <button onClick={authorHandler} style={{ backgroundColor: "blue" }}>
+            <button
+              onClick={authorHandler}
+              style={{ backgroundColor: "#ba7a92" }}
+            >
               Author
             </button>
-          </div>
-        </>
-      )}
-      <div className={classes.actions}>
-        {orderUpDown ? (
-          <button onClick={downHandler} style={{ backgroundColor: "blue" }}>
-            DOWN
-          </button>
-        ) : (
-          <button onClick={downHandler}>DOWN</button>
+          </>
         )}
+        <button onClick={upDownHandler}>
+          {orderUpDown ? (
```

Comment:
Could pull the ternary out by pulling out the `src` and `alt` as well however not sure that would actually be better style so fifty fifty on that one